cartaya/profile-programers/index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./resources/css/style.css">
+    <title>profile-programers</title>
+</head>
+<body>
+    <div class="programer-profile">
+        <img class="programmer-profile_picture" src="./resources/images/sara.png" alt="profile picture">
+        <div class="programer-profile_info">
+            <p class="programer-profile_name lato-font color-font">saray pacheco</p>
+            <a class="programer-profile_mail lato-font" href="mailto:saraypacheco@gmail.com">saraypacheco@gmail.com</a>
+            <div class="programer-profile_challenge-container">
+                <div class="lalala">
```

Comment:
What mean ```the class lalalala ```?. You need to follow the name convention

Reply:
ahhhhhhhh, sorryyy
I didnt get an idea for this class and I was putting that for a moment, and I forgot change it